simplic-user/Simplic.User.UI/Views/UserDetailsView.xaml

```diff
@@ -0,0 +1,86 @@
+﻿<simplic:DefaultRibbonWindow x:Class="Simplic.User.UI.UserDetailsView"
+        xmlns="http://schemas.microsoft.com/winfx/2006/xaml/presentation"
+        xmlns:x="http://schemas.microsoft.com/winfx/2006/xaml"
+        xmlns:simplic="http://schemas.simplic-systems.com/2016/xaml/presentation"
+        xmlns:dlgs="clr-namespace:Simplic.User.UI"
+        xmlns:local="clr-namespace:Simplic.User.UI"
+        xmlns:vm="clr-namespace:Simplic.User.UI"
```

Comment:
Do we need all 3 namespaces?

Reply:
Since you use one namespace per project, I believe that using different prefixes increases the readability of the code. This allows to understand what part of the architecture the class belongs to (which is difficult to do in xaml files).